src/shared/buttons/ViewButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useCallback } from 'react';
+
+import classNames from 'classnames';
+import { useIntl } from 'react-intl';
+
+import { ViewButtonType } from 'shared/buttons/types';
+
+const style = 'cancel-button';
+
+function ViewButton({ onClick, className }: ViewButtonType) {
+  const { formatMessage } = useIntl();
+  const format = useCallback((id: string, values: any = '') => formatMessage({ id }, values), [formatMessage]);
+
+  return (
+    <button
+      type="button"
+      onClick={onClick}
+      className={classNames(style, className)}
+    >
+      <p>{format('app.excel_upload.view_template')}</p>
+    </button>
+  );
+}
+
+export default ViewButton;
```

Comment:
`ViewButton` have the same styles like `CancelButton` - we could just added one file with different name but the same rules and styles and use it instead of `ViewButton` and `CancelButton` - to do later or now if you have time :)

Reply:
yes. we could do this more generic (let's do this when we have time for 'fixes')